nx-X11/programs/Xserver/hw/nxagent/compext/Png.c

```diff
@@ -374,7 +378,7 @@ char *PngCompressData(XImage *image, int *compressed_size)
     png_destroy_write_struct(&png_ptr, &info_ptr);
 
     free(pngCompBuf);
-
+    free(image_index);
```

Comment:
NXCreatePalette{16,32} might store image_index, needs to be checked thoroughly

Reply:
removed